packages/web/src/Client.ts

```diff
@@ -237,8 +237,9 @@ class Client {
     return response?.removeCouponFromCart
   }
 
-  async fetchCart(): Promise<any> {
+  async fetchCart(options: { orderId: string | null }): Promise<any> {
```

Comment:
should this be: `orderId?: string` ?

Reply:
hmm, I did that but then I saw we are following `orderId: string | null` in all other places in this file so kept this. I can change it to `orderId?: string`. Let me know.